src/openloco/windows/PlayerInfoPanel.cpp

```diff
@@ -269,16 +270,16 @@ namespace openloco::ui::PlayerInfoPanel
 
     static void sub_4395C7()
     {
-        CompanyWindow::openFinances(companymgr::get_controlling_id());
+        windows::CompanyWindow::openFinances(companymgr::get_controlling_id());
```

Comment:
This player info panel should probably be in the windows namespace to begin with.